code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -18,6 +18,10 @@
 	var/hair_color = "000"
 	var/hairstyle = "Bald"
 
+	//Gradient color and style
+	var/grad_color = "000"
+	var/grad_style = "None"
```

Comment:
these shouldn't be strings but lmao hair code so eh, not much that can be done w/o rewriting most of hair code